src/lib/pages/backoffice/Providers/ProviderDetails/ProviderDetails.js

```diff
@@ -21,61 +22,74 @@ import {
   Ref,
   Sticky,
 } from 'semantic-ui-react';
-import { LibraryInformation } from './LibraryInformation';
+import { ProviderInformation } from './ProviderInformation';
 
-const DeleteLibraryButton = (props) => {
+const DeleteProviderButton = (props) => {
   return (
     <DeleteButton
       fluid
-      content="Delete library"
+      content="Delete provider"
       labelPosition="left"
       {...props}
     />
   );
 };
 
 class ActionMenu extends React.Component {
-  createRefProps(libraryPid) {
-    const brwReqRefProps = {
-      refType: 'Borrowing Requests',
-      onRefClick: (brwReqPid) =>
-        goTo(ILLRoutes.borrowinRequestDetailsFor(brwReqPid)),
-      getRefData: () =>
-        borrowingRequestApi.list(
-          borrowingRequestApi.query().withLibraryPid(libraryPid).qs()
-        ),
+  createRefProps(providerPid, providerType) {
+    let refType;
+    let path;
+    let query;
+    if (providerType === 'LIBRARY') {
```

Comment:
Since the librarian might be using libraries for ACQs and vendors for BrwReq, I would simply do an `or`.
Pseudocode:
```
borrowingRequestApi.list(
        borrowingRequestApi.query().withProviderPid(providerPid).qs()
      );

OR

orderApi.list(orderApi.query().withProviderPid(providerPid).qs());
```
to be tested :)